doc/classes/EditorPlugin.xml

```diff
@@ -494,6 +494,18 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="add_move_array_element_function">
+			<return type="void" />
+			<param index="0" name="class" type="String" />
+			<param index="1" name="callable" type="Callable" />
+			<description>
+				Allows setting EditorInspectorArray callback to handle the add, remove and change order actions.
```

Comment:
This still doesn't make it very clear (at least to me) what this method does.

Is this an accurate description:

```suggestion
				Allows setting a callback to handle adding, removing and changing the order of elements of an array of the given class in the Godot editor's inspector.
```

Again, since I don't fully understand this API, I may not have that right. However, I don't think mentioning `EditorInspectorArray` is very helpful, and instead trying to describe the situation where this callback is used would be better.